assets/src/edit-story/components/panels/animation/effectChooser.js

```diff
@@ -134,17 +144,85 @@ export default function EffectChooser({
+              onClick={() =>
+                onAnimationSelected({
+                  animation: BACKGROUND_ANIMATION_EFFECTS.ZOOM.value,
+                })
+              }
+            >
+              <ContentWrapper>{__('Zoom', 'web-stories')}</ContentWrapper>
+              <ZoomOutAnimation>{__('Zoom', 'web-stories')}</ZoomOutAnimation>
+            </GridItemFullRow>
+            <GridItem
+              aria-label={__('Pan Left Effect', 'web-stories')}
+              onClick={() =>
+                onAnimationSelected({
+                  animation: BACKGROUND_ANIMATION_EFFECTS.PAN.value,
+                  panDir: DIRECTION.LEFT_TO_RIGHT,
+                })
+              }
+              disabled={disabledTypeOptionsMap[
+                BACKGROUND_ANIMATION_EFFECTS.PAN.value
+              ]?.includes(DIRECTION.LEFT_TO_RIGHT)}
+            >
+              <ContentWrapper>{__('Pan Left', 'web-stories')}</ContentWrapper>
+              <PanLeftAnimation>
+                {__('Pan Left', 'web-stories')}
+              </PanLeftAnimation>
+            </GridItem>
+            <GridItem
+              aria-label={__('Pan Right Effect', 'web-stories')}
+              onClick={() =>
+                onAnimationSelected({
+                  animation: BACKGROUND_ANIMATION_EFFECTS.PAN.value,
+                  panDir: DIRECTION.RIGHT_TO_LEFT,
+                })
+              }
+              disabled={disabledTypeOptionsMap[
+                BACKGROUND_ANIMATION_EFFECTS.PAN.value
+              ]?.includes(DIRECTION.RIGHT_TO_LEFT)}
+            >
+              <ContentWrapper>{__('Pan Right', 'web-stories')}</ContentWrapper>
+              <PanRightAnimation>
+                {__('Pan Right', 'web-stories')}
+              </PanRightAnimation>
+            </GridItem>
+            <GridItem
+              aria-label={__('Pan Up Effect', 'web-stories')}
+              onClick={() =>
+                onAnimationSelected({
+                  animation: BACKGROUND_ANIMATION_EFFECTS.PAN.value,
+                  panDir: DIRECTION.BOTTOM_TO_TOP,
+                })
+              }
+              disabled={disabledTypeOptionsMap[
+                BACKGROUND_ANIMATION_EFFECTS.PAN.value
+              ]?.includes(DIRECTION.BOTTOM_TO_TOP)}
+            >
+              <ContentWrapper>{__('Pan Up', 'web-stories')}</ContentWrapper>
+              <PanBottomAnimation>
+                {__('Pan Up', 'web-stories')}
+              </PanBottomAnimation>
+            </GridItem>
+            <GridItem
+              aria-label={__('Pan Down Effect', 'web-stories')}
+              onClick={() =>
+                onAnimationSelected({
+                  animation: BACKGROUND_ANIMATION_EFFECTS.PAN.value,
+                  panDir: DIRECTION.TOP_TO_BOTTOM,
+                })
+              }
+              disabled={disabledTypeOptionsMap[
+                BACKGROUND_ANIMATION_EFFECTS.PAN.value
+              ]?.includes(DIRECTION.TOP_TO_BOTTOM)}
```

Comment:
Should we disable these animations or not show them? I am assuming they will be enabled in a future PR, but if not, we should just drop.

Reply:
In the designs they had the unavailable options of an animation displayed, but disabled in the effect chooser:
https://www.figma.com/file/NHHMwIfxMnz39NxqkrFb7R/Stories---Post-Stable?node-id=2399%3A4166